src/main/scala/hmda/model/filing/lar/LoanApplicationRegister.scala

```diff
@@ -0,0 +1,127 @@
+package hmda.model.filing.lar
+
+import enums._
+import hmda.model.filing.PipeDelimited
+
+case class LoanApplicationRegister(
+    id: Int = 2,
+    LEI: Option[String] = None,
+    loan: Loan,
+    preapproval: PreapprovalEnum,
+    actionTakenType: ActionTakenTypeEnum,
+    actionTakenDate: Int,
+    geography: Geography,
+    applicant: Applicant,
+    coApplicant: Applicant,
+    purchaserType: PurchaserEnum,
+    rateSpread: String,
+    hoepaStatus: HOEPAStatusEnum,
+    lienStatus: LienStatusEnum,
+    denial: Denial,
+    otherDenialReason: String,
+    loanDisclosure: LoanDisclosure,
+    interestRate: Option[String] = None,
+    prepaymentPenaltyTerm: String,
+    loanToValueRatio: Option[String] = None,
+    introductoryRatePeriod: String,
+    nonAmortizingFeatures: NonAmortizingFeatures,
+    propertyValue: String,
+    manufacturedHomeSecuredProperty: Option[
+      ManufacturedHomeSecuredPropertyEnum] = None,
+    manufacturedHomeLandPropertyInterestEnum: Option[
+      ManufacturedHomeLandPropertyInterestEnum] = None,
+    totalUnits: Option[Int] = None,
+    multiFamilyAffordableUnits: Option[String] = None,
+    applicationSubmission: ApplicationSubmissionEnum,
+    payableToInstitution: PayableToInstitutionEnum,
+    NMLSRIdentified: String,
+    AUS: Option[AutomatedUnderwritingSystem] = None,
+    otherAUS: Option[String] = None,
+    ausResult: Option[AutomatedUnderwritingSystemResult] = None,
+    otherAusResult: Option[String] = None,
+    reverseMortgage: Option[MortgageTypeEnum] = None,
+    lineOfCredit: Option[LineOfCreditEnum] = None,
+    businessOrCommercialPurpose: Option[BusinessOrCommercialBusinessEnum] = None
+) extends PipeDelimited {
+  override def toCSV: String = {
+    val leiStr = LEI match {
+      case Some(lei) => lei
+      case None      => ""
+    }
+
+    val interestRateStr = interestRate match {
+      case Some(rate) => rate
+      case None       => ""
+    }
+
+    val loanToValueRatioStr = loanToValueRatio match {
+      case Some(loanRatio) => loanRatio
+      case None            => ""
+    }
+
+    val manufacturedHomeSecuredPropertyStr =
+      manufacturedHomeSecuredProperty match {
+        case Some(homeSecured) => homeSecured.code
+        case None              => ""
+      }
+
+    val manufacturedHomeLandStr =
+      manufacturedHomeLandPropertyInterestEnum match {
+        case Some(homeLandProperty) => homeLandProperty.code
+        case None                   => ""
+      }
+
+    val totalUnitsStr = totalUnits match {
+      case Some(units) => units.toString
+      case None        => ""
+    }
+
+    val multiFamilyStr = multiFamilyAffordableUnits match {
+      case Some(multifamily) => multifamily
+      case None              => ""
+    }
+
+    val ausStr = AUS match {
+      case Some(aus) => aus.toCSV
+      case None      => ""
```

Comment:
Both here and at line 96 need to account for the missing pipes.  A `None` case should return some set of empty pipes (e.g. `"||||"`).